build.gradle

```diff
@@ -36,10 +37,13 @@ application {
 
 // ### Checkstyle plugin settings
 checkstyle {
-    toolVersion = '10.2'
+    // Latest version at https://github.com/checkstyle/checkstyle/tags
+    toolVersion = '10.3.2'
     // Keep the below file updated along with subsequent versions of Checkstyle (make sure to choose
     // the tag matching the toolVersion above):
-    // https://github.com/checkstyle/checkstyle/blob/checkstyle-10.2/src/main/resources/google_checks.xml
+    // https://github.com/checkstyle/checkstyle/blob/checkstyle-10.3.2/src/main/resources/google_checks.xml
+    // NOTE: It may also be necessary to change "${org.checkstyle.google.suppressionfilter.config}"
```

Comment:
I don't understand this note.

EDIT: ah, I see that this changed in the file itself. Can you clarify this comment to:
- cite something
- indicate that the edit takes place in that file?

Also, was that file updated by just downloading a new one? It might be good to write down a `curl` command here; you could even write `curl ... | sed s|org.checkstyle...|\${config_log}/...|` to automate this away entirely.

Reply:
Added more context per your request. Yeah, I just copy and paste it from GH usually, and I like the idea of automating it with a script like that except the address would need to be changed each time to the match most recent version posted to the Maven repo, and `sed` is a no-go in my Windows environment. :/

Reply:
Ack, sorry, missed this PR with this change. Will include it in my next.